workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/TextController.kt

```diff
@@ -0,0 +1,166 @@
+package com.squareup.workflow1.ui
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+import android.widget.TextView
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.drop
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.suspendCancellableCoroutine
+
+/**
+ * Helper class for working with [EditText]s from workflows.
+ *
+ * ## Usage
+ *
+ * 1. For every editable string in your state, create a property of type [TextController].
+ *    ```
+ *    data class State(val text: TextController = TextController())
+ *    ```
+ * 2. Create a matching property in your rendering type.
+ *    ```
+ *    data class Rendering(val text: TextController)
+ *    ```
+ * 3. In your `render` method, copy each [TextController] from your state to your rendering:
+ *    ```
+ *    return Rendering(state.text)
+ *    ```
+ * 4. In your `LayoutRunner`'s `showRendering` method, call [control] and pass your [EditText].
```

Comment:
```suggestion
 * 4. In your `LayoutRunner`/`AndroidViewRendering`/`Screen`'s `showRendering` method, call [control] and pass your [EditText].
```

Reply:
`AndroidViewRendering` doesn't have a `showRendering` method, and `Screen` doesn't exist. How about:

```
4. In your view code's `showRendering` method…
```